lambda-buffers-compiler/src/LambdaBuffers/Compiler/KindCheck/Judgement.hs

```diff
@@ -8,5 +9,17 @@ import Prettyprinter (Pretty (pretty), (<+>))
 newtype Judgement = Judgement {getJudgement :: (Context, Type, Kind)}
   deriving stock (Show, Eq)
 
+jType :: Lens' Judgement Type
+jType = lens from to
+  where
+    from = (\(_, x, _) -> x) . getJudgement
+    to (Judgement (c, _, k)) t = Judgement (c, t, k)
+
+jKind :: Lens' Judgement Kind
+jKind = lens from to
+  where
+    from = (\(_, _, k) -> k) . getJudgement
+    to (Judgement (c, t, _)) k = Judgement (c, t, k)
```

Comment:
Just make Judgement a data with the record syntax and use makeLenses, this is unnecessary code bloat.